modules/es-extensions/publisher/extensions/app/greg-associations/themes/default/partials/association-base.hbs

```diff
@@ -54,7 +54,7 @@
                     {{/each}}
                 {{else}}
                     <div class="emptyAssets-MsgDiv">
-                        <p class="emptyAssets">We are sorry! we couldn't find any associations for {{assets.name}}...</p>
+                        <p class="emptyAssets">We are sorry! we couldn't find any associations for this asset.</p>
                     </div>
```

Comment:
Hi Denuwanthi,

In this case asset name is 'info'? shall we bold the name instead of removing it?

Thanks,
Thushara.